src/services/bria/index.ts

```diff
@@ -264,6 +280,26 @@ export const NewOnChainService = (): INewOnChainService => {
     }
   }
 
+  const rebalanceToColdWallet = async (
+    amount: BtcPaymentAmount,
+  ): Promise<PayoutId | OnChainServiceError> => {
+    try {
+      const request = new SubmitPayoutRequest()
+      request.setWalletName(briaConfig.hotWalletName)
+      request.setPayoutQueueName(briaConfig.coldStorage.hotToColdRebalanceQueueName)
+      request.setDestinationWalletName(briaConfig.coldStorage.walletName)
```

Comment:
cold storage address is generated internally in bria?

Reply:
Yes if you use `setDestinationWalletName` then an address is generated internally. https://github.com/GaloyMoney/bria/blob/897b4faf65183a49524d4d8bb7769c5d15741ca3/proto/api/bria.proto#L342-L345